src/common/vscode/tree/ExpandedErrorTreeItem.ts

```diff
@@ -0,0 +1,12 @@
+import * as vscode from "vscode";
+import TreeItem from "./TreeItem";
+
+export default class ExpandedErrorTreeItem extends TreeItem {
+    public constructor(title: string, private message: string) {
```

Comment:
Sorry if this is a newbie question 😄 I couldn't understand how `message` is set to class instance. Could you explain shortly or redirect me somewhere so that I could understand it better!

Reply:
```
public constructor(title: string, private message: string) {
    super(title, vscode.TreeItemCollapsibleState.Expanded);
}
```

is the same as

```
private message: string;

public constructor(title: string, message: string) {
    super(title, vscode.TreeItemCollapsibleState.Expanded);
    this.message = message;
}
```

keyword "private" in the first constructor does it

Reply:
Thank you! I didn't know about this shorthand.